src/query/test/executor/transform.go

```diff
@@ -51,34 +51,36 @@ type SinkNode struct {
 
 // Process processes and stores the last block output in the sink node
 func (s *SinkNode) Process(_ *models.QueryContext, ID parser.NodeID, block block.Block) error {
-	iter, err := block.SeriesIter()
+	iter, err := block.StepIter()
```

Comment:
nit: Move this above `iter, err := block.StepIter()` so that it more naturally flows?

i.e. 

```go
s.Meta = block.Meta()
iter, err := block.StepIter()
if err != nil {
  return err
}
```

Reply:
Nice sounds good